packages/path_provider/path_provider_foundation/test/path_provider_foundation_test.dart

```diff
@@ -20,6 +21,9 @@ void main() {
 
   group('PathProviderFoundation', () {
     late PathProviderFoundation pathProvider;
+    late PathProviderFoundation iosPathProvider;
+    late PathProviderFoundation macosPathProvider;
```

Comment:
Per discussion in chat, I would eliminate all three of these so that the tests fixture is less stateful. Every test would then start with:

```
pathProvider = PathProviderFoundation();
```

in the standard case, or:

```
pathProvider = PathProviderFoundation(platform: someMockPlatform);
```

when necessary.